packages/expo-updates/src/UseUpdatesUtils.ts

```diff
@@ -86,5 +73,8 @@ export const reduceUpdatesStateFromContext = (
     downloadedUpdate,
     checkError: context.checkError,
     downloadError: context.downloadError,
+    lastCheckForUpdateTimeSinceRestart: context.isChecking
+      ? new Date()
```

Comment:
should we only update this if it previously was `isChecking: false` and is about to become `isChecking: true`? Otherwise, all state change events that occur while `isChecking` is true will cause this date to update.

Reply:
The native layer only fires `isChecking=true` events during app startup and when `checkForUpdateAsync()` is called, so if there are multiple such events, the last one should win. So the current implementation is ok for now.

Probably, what we should do to be completely correct is to add the last check dateTime to the native context, and update it not on the `check` event, but on `checkComplete` and `checkError` events. Then JS should just read the time from that.

Reply:
It may fire other events in between when isChecking is set to true and when it is again set to false. In this case it'll update the date for each of those events.